extra_docs/dataset/README.md

```diff
@@ -0,0 +1,269 @@
+# Datasets Module
+A Dataset is used to manipulate records.
```

Comment:
I would add a bit more description, i.e:
```
A Datasets are used for manipulating user data which is organized as records.  
You can create, update, fetch and delete any records. 
It's also possible to query records using conditions like `isEqualTo`, `isGreaterThen`, etc.
```

Reply:
Maybe we can use next : Datasets is SQL/NoSQL data storage where user data can be stored with schema and schemaless way. 
You can create, update, fetch and delete any records. 
It's also possible to query records using conditions like `isEqualTo`, `isGreaterThen`, etc.

Reply:
`where data can be stored under schema/schemaless ways. `
this sentence a bit unclear to me, I would explain it in a more detailed way. maybe:
`where data is stored either using a strict schema or without it (schemaless). `

Reply:
agreed :+1: